src/components/WalletConnect.vue

```diff
@@ -36,6 +36,7 @@ import { defineComponent } from "vue";
 import { WalletType } from "../types/enum.types";
 import { WebMode } from "@algo-builder/web";
 import WalletStore from "../store/WalletStore";
+import { searchForAssets } from "../indexer";
 declare var AlgoSigner: any; // eslint-disable-line
```

Comment:
I have a question here. I tried below import 
`import { AlgoSigner } from "@algo-builder/web/build/algo-signer-types"`

but while trying to use AlgoSigner.connect() method I am getting an error that connect method is not defined. I tried checking other projects they were also using `any` for AlgoSigner. Is there another data type for AlgoSigner which I am missing?

Reply:
a) The import is wrong. You shouldn't import from build. Do `import { types } from "@algo-builder/web"`. Then use `types.AlgoSigner`

b) It throws an error because `connect()` method is not implemented in [this](https://github.com/scale-it/algo-builder/blob/master/packages/web/src/algo-signer-types.ts#L147) type definition. It should be. 

If you want, you can do it as a separate task.

Reply:
Seems like `types.AlgoSigner` also does not exist. Getting error AlgoSigner type does not exist.

I made below code change:
`import { types } from "@algo-builder/web";
declare var AlgoSigner: types.AlgoSigner;`

I am using `"@algo-builder/web": "^3.2.0",` as package dependency. Is there any specific version in which this is available?

Reply:
@rmeena840 see the develop branch. You can use github commit as a dependency version, instead of the release. In that case you should use the same version in all: web, runtime and algob in package.json

Reply:
Also, I agree with @ratik21 - we want to limit to minimum the eslint disable. You should always justify that addition.

Reply:
I checked the documentation and their documentation itself uses `any` as a datatype.
https://developer.algorand.org/solutions/wagering-dapp-with-a-serverless-vue-algosigner-frontend/#algosigner-api

I think we have to go with `any` here.

Reply:
So it exists in window object. See below attached image.
![Screenshot 2022-03-31 at 7 25 50 PM](https://user-images.githubusercontent.com/17853389/161072724-65dffdbe-2064-417e-b7ea-08bb6f375fb6.png)